geojson-strip-html/index.js

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Add a shebang so this can be executed directly
```suggestion
#!/usr/bin/env node
/*
```